packages/scandipwa/src/component/MyAccountCreateAccount/MyAccountCreateAccount.component.js

```diff
@@ -96,7 +98,7 @@ export class MyAccountCreateAccount extends PureComponent {
 
         return (
             <fieldset block="MyAccountOverlay" elem="Legend">
-                <legend>{ __('Sign-Up Information') }</legend>
+                <legend block="MyAccountOverlayLegend">{ __('Sign-Up Information') }</legend>
```

Comment:
Please make this `MyAccountOverlay-Legend`.
Now, judging by the classname, one might think that this is a component (which it obviously is not).

Reply:
Agree, but this will interfere with the parent fieldset classname that is `MyAccountOverlay-Legend` too.
Also i see that upper function `renderCreateAccountPersonalInfoFields` returns same layout, but fieldset are declared differently.
And i don't see a use of the new classname `MyAccountOverlayLegend`. I'm not sure if it is needed.
I'd suggest to remove unused classname and make classname declaration consistent between the `renderCreateAccountPersonalInfoFields` and the `renderCreateAccountSignUpInfoFields`.

Reply:
Then I propose to figure out some other name, consistent with the other class namings.

Block - MyAccountOverlay
Elem - ? // maybe LegendTitle or something like that.

Though it's surprising that fieldset's item name is "legend". Maybe this is a mistake?

Reply:
Changed the overall structure. This should do 🙏

Reply:
And still, do you really need that class here? What it's purpose?
![image](https://user-images.githubusercontent.com/73945186/121225235-fb7cb800-c891-11eb-8daf-b448fe0c1dbb.png)